runauto_nag_hourly.sh

```diff
@@ -64,5 +64,9 @@ python -m auto_nag.scripts.mismatch-priority-tracking-release
 python -m auto_nag.scripts.mismatch-priority-tracking-beta
 python -m auto_nag.scripts.mismatch-priority-tracking-nightly
 
+# Move info (signatures, product/component) from/to bugs & their dups
+# Pretty common
+python2 -m auto_nag.scripts.duplicate
```

Comment:
duplicate crash ?

Reply:
what do you mean ?

Reply:
duplicate is too generic.
so, I would prefer that we make it specific to crash
renaming for duplicate_crash for example

Reply:
yeah you're right, but I'm moving product/info too (from original to dup)
So duplicate_bug maybe ?

Reply:
Maybe `copy_duplicate_info`?

Reply:
wfm, @sylvestre ?

Reply:
Works for me